sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Config/EventHubOptions.cs

```diff
@@ -88,6 +111,55 @@ public int MaxBatchSize
         /// </summary>
         public InitialOffsetOptions InitialOffsetOptions { get; }
 
+        /// <inheritdoc cref="EventProcessorOptions.TrackLastEnqueuedEventProperties"/>
+        public bool TrackLastEnqueuedEventProperties
+        {
+            get => EventProcessorOptions.TrackLastEnqueuedEventProperties;
+            set => EventProcessorOptions.TrackLastEnqueuedEventProperties = value;
+        }
+
+        /// <inheritdoc cref="EventProcessorOptions.PrefetchCount"/>
+        public int PrefetchCount
+        {
+            get => EventProcessorOptions.PrefetchCount;
+            set => EventProcessorOptions.PrefetchCount = value;
+        }
+
+        /// <inheritdoc cref="EventProcessorOptions.PrefetchSizeInBytes"/>
+        public long? PrefetchSizeInBytes
+        {
+            get => EventProcessorOptions.PrefetchSizeInBytes;
+            set => EventProcessorOptions.PrefetchSizeInBytes = value;
+        }
+
+        /// <inheritdoc cref="EventProcessorOptions.Identifier"/>
+        public string Identifier
+        {
+            get => EventProcessorOptions.Identifier;
+            set => EventProcessorOptions.Identifier = value;
+        }
+
+        /// <inheritdoc cref="EventProcessorOptions.MaximumWaitTime"/>
+        public TimeSpan? MaximumWaitTime
+        {
+            get => EventProcessorOptions.MaximumWaitTime;
```

Comment:
Is it worth having a copy of EventProcessorOptions here? Can we create an instance when starting the processor?

Reply:
The benefit is that we can use the property validation without copying and pasting.

Reply:
Doesn't sounds like a good reason to me especially considering a part of options (initial position) are applied later via a separate code path. Creating processor options on-demand would-ve fixed that.'

Up to you.